std/math.d

```diff
@@ -6922,8 +6922,10 @@ real fdim(real x, real y) @safe pure nothrow @nogc { return (x > y) ? x - y : +0
 
 /**
  * Returns the larger of x and y.
+ *
+ * If one of the arguments is a NaN, the other is returned.
  */
-real fmax(real x, real y) @safe pure nothrow @nogc { return x > y ? x : y; }
+real fmax(real x, real y) @safe pure nothrow @nogc { return core.stdc.math.fmaxl(x, y); }
```

Comment:
Could we use isNaN here to allow the compiler to inline the tests? (Not a problem for compilers that treat the extern math functions as builtins).

Reply:
Done.